nsca_client/models/nsca_server.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# © 2015 ABF OSIELL <http://osiell.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
```

Comment:
@StefanRijnhart It is fine by me not to spot them if it will be taken care somewhere else. I know that some work is ongoing for deeper formal check: is what you meant?